### ISSUE TITLE
How to render Json-tree-view initially collapsed

### ISSUE THREAD

**Issue by ghost** [bazh/jquery.json-view#3]
Hello,
Is there a way to render the json-tree-view initially collapsed?
I tried simply adding 'true' as a parameter to 
item.prepend(collapser( true ));

That changes the direction of the triangle because the collapsed class gets added,
but the ul children aren't hidden.
That only seems to be done in the on click handler.
Even doing the above, that would require me to change your source.
Is there a way to do it programmatically from my code?

Thanks
David

**Comment by ericleb010** [comment]
Yep, this would be a nice feature to have when the JSON data size is significant.

**Comment by dscoduc** [comment]
What about just adding this right after the jsonView call:

`$('.collapser').click();`
